boilerplate/fullstack/sveltekit/src/routes/auth/[...anything]/+page.svelte

```diff
@@ -0,0 +1,23 @@
+<script lang="ts">
```

Comment:
Long story short, because of how svelte's routing works. Since it's using file-based routing, we can't really do things like catch-all routes (i.e. `*` route which resolves to an Auth components, similar to what we're doing in Vue). So there has to be something that would intercept a request or a callback at all those levels in the auth route. 

BTW, we're already using a similar approach with Astro (which suffers from the same issue).